stdlib/LinearAlgebra/src/blas.jl

```diff
@@ -664,13 +664,19 @@ for (fname, elty) in ((:dgemv_,:Float64),
                 throw(DimensionMismatch("the transpose of A has dimensions $n, $m, X has length $(length(X)) and Y has length $(length(Y))"))
             end
             chkstride1(A)
+            lda = stride(A,2)
+            lda >= max(1, size(A,1)) || error("`stride(A,2)` must be at least `max(1, size(A,1))`")
+            sX = stride(X,1)
+            pX = pointer(X, sX > 0 ? firstindex(X) : lastindex(X))
+            sY = stride(Y,1)
+            pY = pointer(Y, sY > 0 ? firstindex(X) : lastindex(X))
```

Comment:
Is `pointer` safe to use here or do I need to use `Ref` to prevent the arrays from getting GC’d?

Reply:
you should wrap in `GC.@preserve X Y begin ... end`

Reply:
Done!